### PR TITLE
EZP-28029: As a frontend developer I would like to use BEM naming convention for styling our components

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZP-28029
> Requires: https://github.com/ezsystems/repository-forms/pull/163

### CODE REVIEW

**Comment by dew326** [comment]
@sunpietro frontend ready for review: https://github.com/ezsystems/ezplatform-admin-ui/pull/22/commits/4e74e4cd0780a5a520d9fff32e4fd9ced8fde484

**Comment by lserwatka** [comment]
@dew326 @webhdx what else is needed here to get ride of WIP?

**Comment by dew326** [comment]
For the frontend part PR is ready to merge.

**Comment by webhdx** [comment]
This PR is ready for review. We have to merge https://github.com/ezsystems/repository-forms/pull/163 first.

File: src/bundle/Resources/config/services/content_edit.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    ezrepoforms.view_templates_listener:
+        class: "%ezrepoforms.view_templates_listener.class%"
+        tags:
+            - { name: kernel.event_subscriber }
+        calls:
+            - [setViewTemplate, ['EzSystems\RepositoryForms\UserRegister\View\UserRegisterFormView', "$user_registration.templates.form$"]]
+            - [setViewTemplate, ['EzSystems\RepositoryForms\UserRegister\View\UserRegisterConfirmView', "$user_registration.templates.confirmation$"]]
+            - [setViewTemplate, ['EzSystems\RepositoryForms\Content\View\ContentEditView', "$content_edit.templates.edit$"]]
+            - [setViewTemplate, ['EzSystems\RepositoryForms\Content\View\ContentCreateDraftView', "$content_edit.templates.create_draft$"]]
+            - [setPagelayout, ["$pagelayout$"]]
```
Comment (alongosz) [inline src/bundle/Resources/config/services/content_edit.yml]: Please use single quotes in new yaml files.

**Pull request merged**